understanding/understanding-techniques.html

```diff
@@ -100,40 +100,40 @@ <h2>Failures</h2>
 		</section>
 		<section>
 			<h2>Other Techniques</h2>
-			<p>In addition to the techniques in W3C's <a href="https://www.w3.org/WAI/WCAG21/Techniques/">Techniques for WCAG 2.1</a> document, <em>there are other ways to meet WCAG success criteria</em>. W3C's techniques are not comprehensive and may not cover newer technologies and situations.</p>
-			<p><em>Web content does not have to use W3C's published techniques in order to conform to WCAG 2.1.</em><em>(See also <a href="understanding-techniques-informative">Techniques are Informative</a> above.)</em></p>
-			<p>Content authors can develop different techniques. For example, an author could develop a technique for HTML5, <a href="https://www.w3.org/WAI/intro/aria">WAI-ARIA</a>, or other new technology. Other organizations may develop sets of techniques to meet WCAG 2.1 success criteria.</p>
+			<p>In addition to the techniques in W3C's <a href="../Techniques/">Techniques</a> document, <em>there are other ways to meet WCAG 2 success criteria</em>. W3C's techniques are not comprehensive and may not cover newer technologies and situations.</p>
+			<p><em>Web content does not have to use W3C's published techniques in order to conform to WCAG 2. (See also <a href="understanding-techniques-informative">Techniques are Informative</a> above.)</em></p>
+			<p>Content authors can develop different techniques. For example, an author could develop a technique for HTML5, <a href="https://www.w3.org/WAI/intro/aria">WAI-ARIA</a>, or other new technology. Other organizations may develop sets of techniques to meet WCAG 2 success criteria.</p>
 			<p>Any techniques can be sufficient if:</p>
 			<ul>
 				<li>
 					<p>they satisfy the success criterion, and</p>
 				</li>
 				<li>
-					<p> all of the <a href="conformance">WCAG 2.1 conformance requirements</a> are met.</p>
+					<p> all of the <a href="conformance">WCAG 2 conformance requirements</a> are met.</p>
 				</li>
 			</ul>
 			<section>
 				<h3>Submitting Techniques</h3>
-				<p>The WCAG Working Group encourages people to submit new techniques so that they can be considered for inclusion in updates of the <a href="https://www.w3.org/WAI/WCAG21/Techniques/">Techniques for WCAG 2.1</a> document. Please submit techniques for consideration using the <a href="https://www.w3.org/WAI/GL/WCAG20/TECHS-SUBMIT/">Techniques Submission Form</a>.</p>
+				<p>The WCAG Working Group encourages people to submit new techniques so that they can be considered for inclusion in updates of the <a href="../Techniques/">Techniques</a> document. Please submit techniques for consideration to the <a href="https://github.com/w3c/wcag/">WCAG source repository</a>.</p>
```

Comment:
Is the common term for the WCAG GitHub repository the 'WCAG source repository'? This is fine if that is the common name, I just figured I'd double-check that this is a consistent term.

Reply:
honestly, not sure if there's an "official" term for it

Reply:
we could also use "WCAG repository on GitHub" or similar

Reply:
My recommendation:
`WCAG GitHub repository`

The URL points to GitHub anyhow, so there's not much benefit to keeping the language generic. A change in URL would be needed regardless, if we ever change technology.

Also, "WCAG GitHub repository" is searchable, and less likely to be mistaken for some other WCAG source-type thing we might have.

Reply:
@patrickhlauke I'm cool with your suggestion of "WCAG repository on GitHub". This may not need to be thought through too much, as long as it clarifies what we're referring to 👍